### PR TITLE
Mapview license and user images on single event

### PR DESCRIPTION
**Pull request by idosavion**
Added mapview and rearranged the whole activity's layout

### CODE REVIEW

File: app/src/main/java/com/huji/foodtricks/buddies/RSVPListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        ArrayList<String> names = new ArrayList<>();
+        ArrayList<String> RSVPs = new ArrayList<>();
+        ArrayList<UserModel> userModels = new ArrayList<>();
+        for (String userId : usersMap.keySet()) {
+            if (eventAttendanceProvider.getInvitees().keySet().contains(userId)) {
+                UserModel userModel = usersMap.get(userId);
+                names.add(userModel.getUserName());
+                userModels.add(userModel);
+                EventAttendanceProvider.RSVP rsvp = eventAttendanceProvider.getUserRSVP(userId);
+                switch (rsvp) {
```
Comment (mharsat) [inline app/src/main/java/com/huji/foodtricks/buddies/RSVPListAdapter.java:82]: Why not stick to the shorter terminology like at the bottom of the screen? (Going/Coming, Not Going/Not Coming, Maybe, Unknown/Invited/Didn't Respond)

File: app/src/main/res/layout/layout.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (mharsat) [inline app/src/main/res/layout/layout.xml]: please give the file some indicative name

**Review by mharsat — DISMISSED** [review]
Huge improvement awesome work!!!!!
Just
- [ ] Fix time to match format in other parts of the app
- [ ] I suggest making it possible to click the map and pass a location intent (passing a location intent gives the user the option to open his favorite maps app for navigation)

File: app/src/main/java/com/huji/foodtricks/buddies/RSVPListAdapter.java (hunk 1 above)
Comment (idosavion) [reply app/src/main/java/com/huji/foodtricks/buddies/RSVPListAdapter.java:82]: chose going/not going/maybe

File: app/src/main/res/layout/layout.xml (hunk 2 above)
Comment (idosavion) [reply app/src/main/res/layout/layout.xml]: my bad - fixed

**Comment by idosavion** [comment]
About the time format - it's fine, the time of the event was 8AM
about going into the map application, when clicking the location itself two small buttons appear: one for navigating and one for going into the map application,
which I think is better so I'm keeping it that way for now 🍳

**Review by mharsat — APPROVED** [review]

**Pull request merged**